src/utils/create_component_file/index.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
```

Comment:
This file should not be a part of Git repository, it is generated dynamically.

Reply:
Removed (added to .gitignore)